### PR TITLE
Added dev watermark underlay to lighthouse pages

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
watermark class is added to body to set a repeating background image.

JS checks manifest and looks to see if extension name has the word 'Development' in it, and only adds css class if found.

This addresses a concern from state regarding dev versions of code used in production environments unknowingly

### CODE REVIEW

**Comment by OSPFNeighbour** [comment]
screenshot 2016-09-22 15 22 15

File: pages/scripts/stats.js [hunk 1]
```diff
@@ -18,6 +18,29 @@ var timeoverride = null;
 var timeperiod;
 var unit = null;
 
+chrome.manifest = (function() {
```
Comment (sdunster) [inline pages/scripts/stats.js]: please can we put this in a shared library? just like shared_job_code.js etc above?

File: pages/scripts/stats.js [hunk 2]
```diff
@@ -18,6 +18,29 @@ var timeoverride = null;
...
+
+    var manifestObject = false;
+    var xhr = new XMLHttpRequest();
+
+    xhr.onreadystatechange = function() {
+        if (xhr.readyState == 4) {
+            manifestObject = JSON.parse(xhr.responseText);
+        }
+    };
+    xhr.open("GET", chrome.extension.getURL('/manifest.json'), false);
```
Comment (sdunster) [inline pages/scripts/stats.js]: I suggest converting this to use `$.get` instead:
```
$.get(chrome.extension.getURL('/manifest.json'), function(data) {
  // data is now the JSON from manifest
});
```

File: pages/styles/include/common.css [hunk 3]
```diff
@@ -1,7 +1,13 @@
 @import url("node_modules/bootstrap/dist/css/bootstrap.css");
+html {
+height: 100%;
```
Comment (sdunster) [inline pages/styles/include/common.css]: indent

**Comment by OSPFNeighbour** [comment]
Yeah I will.

File: pages/scripts/stats.js (hunk 1 above)
Comment (OSPFNeighbour) [inline pages/scripts/stats.js]: Done

**Pull request merged**